Modules/_threadmodule.c

```diff
@@ -1102,6 +1102,24 @@ thread_run(void *boot_raw)
     // to open the libgcc_s.so library (ex: EMFILE error).
 }
 
+static PyObject *
+thread_daemon_threads_allowed(PyObject *module, PyObject *Py_UNUSED(ignored))
+{
+    PyInterpreterState *interp = _PyInterpreterState_Get();
+    if (interp->feature_flags & Py_RTFLAGS_DAEMON_THREADS) {
+        Py_RETURN_TRUE;
+    }
+    else {
+        Py_RETURN_FALSE;
+    }
+}
+
+PyDoc_STRVAR(daemon_threads_allowed_doc,
+"daemon_threads_allowed()\n\
+\n\
+Return True if daemon threads are allowed in the current interpreter,\n\
```

Comment:
Why does this need to be a function? Just to prevent people from attempting to set it?

Reply:
Daemon threads are implemented exclusively in threading.py, so we need a way to access the info from Python.

Reply:
If you mean why not an attribute, it's because I was thinking of it as an interpreter setting, rather than state of the _thread module.